hazelcast-code-generator/src/main/java/com/hazelcast/client/protocol/generator/CodeGenerationUtils.java

```diff
@@ -56,6 +56,30 @@
         put("com.hazelcast.map.impl.SimpleEntryView", "SimpleEntryView");
     }};
 
+    private static final Map<String, String> JAVA_TO_NODE_TYPES = new HashMap<String, String>() {{
```

Comment:
actually, this parameters are used for method calls. Like read`Int32` or read`Boolean`...